### PR TITLE
To-Do App 

### PR DESCRIPTION
**Pull request by Bondok6**
# Hello there 🙋‍♂️,

### Thanks for taking the time to review my project.

<hr>

## Changes 🧾

- Create React App (CRA).
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called `components` that contain all components files.
- The user can `Add`, `Delete`, and mark a task as `completed`.
- Install the `UUID` library to generate a unique `id` for each task.
- Use [gh-pages](https://www.npmjs.com/package/gh-pages).
- Update `README.md` file.
- Use `Gitflow`.

Gif found on CodeReviewGifs.com hosted on Giphy.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
...
+Then `npm install` to install packages
+
+Finally `npm start`
+
+## Authors
+
+👤 **Kyrillos Hany**
+
+- GitHub: [@Bondok6](https://github.com/Bondok6)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (carloshs1994) [inline README.md]: Hi! I think it would be better to put a link to your real LinkedIn account.

**Review by carloshs1994 — CHANGES_REQUESTED** [review]
# Hi @Bondok6  👋🏻 

Almost there...


You've done a great job so far! Let's highlight the best approaches within this project:

- ✅ Use of ES6 syntax creating a clean codebase.
- ✅ Successfully display the To-Do list.
- ✅ Proper use of variable names.
- ✅ Components are working as intended.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

------

File: README.md (hunk 1 above)
Comment (Bondok6) [reply README.md]: oh, I think I wrote the wrong link, Thank you for the review 👌

**Comment by Bondok6** [comment]
## HI @carloshs1994 🙋‍♂️

#### Thank you for the review. I have made the changes and updated the `README.md` file. So when you have time, could you please re-review my code.

**Review by carloshs1994 — APPROVED** [review]
# Hi @Bondok6   🔥🔥, 

### Your feature is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀

> You did an amazing job here 💯 keep it up👍

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bondok6** [comment]
## Thank you @carloshs1994 🎉🙏

**Pull request merged**